index.html

```diff
@@ -10,9 +10,15 @@
 </head>
 <body>
```

Comment:
- [ ]  [OPTIONAL] Hi I can notice that your design is not aligned to the center on the mobile version. Please take the time to ensure that it is aligned in the center in the mobile version.
![image](https://user-images.githubusercontent.com/18730106/144301048-8fec34de-1fde-4044-9de5-57391967deac.png)